TransferMethod/Tests/SelectTransferMethodTypePresenterTests.swift

```diff
@@ -51,8 +51,6 @@ class SelectTransferMethodTypePresenterTests: XCTestCase {
                       "The transferMethodTypeTableViewReloadData should be performed")
 
         XCTAssertNotNil(HyperwalletInsights.shared, "HyperwalletInsights should be initialized")
-        XCTAssertTrue(hyperwalletInsightsMock.didTrackImpression,
-                      "HyperwalletInsights.trackImpression should be called")
```

Comment:
If this is expected behavior I would say rather than removing assert true, replace it with assert false

Reply:
ok, fixed